airbyte-cdk/python/unit_tests/sources/file_based/in_memory_files_source.py

```diff
@@ -25,18 +26,20 @@ def __init__(
             file_type,
             availability_strategy: AvailabilityStrategy,
             discovery_policy: AbstractDiscoveryPolicy,
-            validation_policies: AbstractSchemaValidationPolicy,
+            validation_policies: Dict[str, AbstractSchemaValidationPolicy],
             parsers: Dict[str, FileTypeParser],
             stream_reader: AbstractFileBasedStreamReader,
+            catalog: Optional[Dict[str, Any]],
```

Comment:
It's not clear how this will be initialized in a real sync. Will that be done as part of https://github.com/airbytehq/airbyte/issues/27022 ?

Reply:
Yes that's right.